components/list/list.js

```diff
@@ -143,6 +197,13 @@ class List extends PageableMixin(SelectionMixin(LitElement)) {
 		`;
 	}
 
+	updated(changedProperties) {
+		super.updated(changedProperties);
+		if (changedProperties.has('breakpoints') && changedProperties.get('breakpoints') !== undefined) {
+			this.resizedCallback(this.offsetWidth, true);
+		}
```

Comment:
The intention of this is to only call `resizedCallback` not on the initial setting of `breakpoints` since it would be getting called anyways by the `ResizeObserver`